javalin/src/main/java/io/javalin/plugin/rendering/vue/JavalinVue.kt

```diff
@@ -60,22 +65,23 @@ object JavalinVue {
         |        state: ${JavalinJson.toJson(state ?: stateFunction(ctx))}
         |    }""".trimMargin() + "\n</script>\n"
 
-    internal fun setRootDirPathIfUnset(ctx: Context) {
-        vueDirPath = vueDirPath ?: if (ctx.isLocalhost()) Paths.get("src/main/resources/vue") else PathMaster.classpathPath("/vue")
+    internal fun setRootDirPathIfUnset() {
+        vueDirPath = vueDirPath ?: if (isDev) Paths.get("src/main/resources/vue") else PathMaster.classpathPath("/vue")
     }
 
     private fun String.replaceWebjarsWithCdn() =
-            this.replace("@cdnWebjar/", if (useCdn) "https://cdn.jsdelivr.net/webjars/org.webjars.npm/" else "/webjars/")
+            this.replace("@cdnWebjar/", if (isDev) "/webjars/" else "https://cdn.jsdelivr.net/webjars/org.webjars.npm/")
```

Comment:
Does this mean that `@cdnWebjar` is looking for localhost files when `isDev` is true?

Reply:
Yes. When `isDev` is `true`, webjars will be fetched from Javalin, not CDN.
(There is no change here from how it was before).

Reply:
Agree that there was also an if/else here before. I realise now that I would be slightly surprised if something that said `cdn` would fetch something locally, but after considering it for a second I would be glad for the speed-up fetching from localhost gives while developing. Don't change it on my account.

Reply:
The name is contradictory, since `cdn` and `webjars` are mutually exclusive options.
`@cdnIfNotDevAndWebjarsIfDev` isn't quite a catchy thought 😄